core-bundle/assets/styles/backend-search/hit.pcss

```diff
@@ -0,0 +1,46 @@
+.tl_search_hit {
+    &:not(:last-of-type) {
```

Comment:
Just out of curiosity: why not `.tl_search_hit:not(:last-of-type) {`?

Reply:
Would work as well, was out of habit when removing other styles I had in tl_search_hit.

Reply:
See 37d277c20e67cc637b27a3d42ec78a31871b369b